### DISCUSSION TITLE
What are your goals and expectations? How would you rate your current experiences?

### DISCUSSION THREAD

**Discussion by davidgitit** [Drook-tech/Bootcamp#11] [Q&A]
What are your goals and expectations for this Bootcamp? 

How would you rate your current experiences?

Azure DevOps?
Jenkins
GitLab
JIRA
Other?

**Comment by davidgitit** [comment]
What are your goals and expectations for this Bootcamp?

To learn Valet!

How would you rate your current experiences?

Azure DevOps?   👍 
Jenkins. 👎 
GitLab 🆗 
JIRA 💯 
Other?

**Comment by craigmoyle** [comment]
What are your goals and expectations for this Bootcamp?

To get a deeper understanding of the more advanced features of GitHub

How would you rate your current experiences?

Azure DevOps? 👍
Jenkins 👍
GitLab 👎
JIRA 👍
Other? Bitbucket

**Comment by tw3lveparsecs** [comment]
What are your goals and expectations for this Bootcamp?
Learn more about migrations and Valet tool

How would you rate your current experiences?

Azure DevOps? 👍 
Jenkins 👎 
GitLab 👎 
JIRA 👎 
Other?

**Comment by william-liebenberg** [comment]
What are your goals and expectations for this Bootcamp?

Learn more advanced features of GitHub :) and then apply to customer engagements.

How would you rate your current experiences?

Azure DevOps 💯 
Jenkins 0️⃣  
GitLab 🆗 
JIRA 🆗 
BitBucket 🆗 
Other?

**Comment by wicksipedia** [comment]
What are your goals and expectations for this Bootcamp?

Get to know some GHAS

How would you rate your current experiences?

Azure DevOps? 💯
Jenkins 😒
GitLab 😒
JIRA 🆗
Other?
- Bitbucket 🆗
- Teamcity 🤮

**Comment by Saraquin** [comment]
**What are your goals and expectations for this Bootcamp?**

Better understanding of direct deploy to Azure.

**How would you rate your current experiences?**

Azure DevOps 👍 
Jenkins 👎 
GitLab 👎 
JIRA 🆗 
Other 🚫

**Comment by s-aqua** [comment]
What are your goals and expectations for this Bootcamp?

Learning migrations 

How would you rate your current experiences?

Azure DevOps? 👍
Jenkins 👎
GitLab 👎
JIRA 👎
Other?
